src/store/reducers.js

```diff
@@ -155,5 +165,6 @@ export const ComponentReducers = {
   accessLoading: accessLoadingReducer,
   access: accessReducer,
   meteringCount: meteringReducer,
-  assetPlans: assetPlansReducer
+  assetPlans: assetPlansReducer,
+  campaignSubscriptions: campaignSubscriptionReducer
```

Comment:
As this is the root reducer, it's better to name it more specifically so that people can understand easily what does this reducer do. `AccessTypeCampaignSubscriptions` like that.